### ISSUE TITLE
Example Plugin (SwagPluginSystem) causes White Screen for Shopware 5.7

### ISSUE THREAD

**Issue by BenRoe** [shopware/devdocs#1094]
I download the zip file from the https://github.com/shopware/devdocs/blob/master/source/developers-guide/example-plugin/index.md and put the `SwagPluginSystem` folder in `custom/plugins`.
After activating the plugin in the Plugin-Manager, i get a white screen on the shop page.

**Comment by mitelg** [comment]
hey @BenRoe 

thanks for bringing this up, but I cannot reproduce this. Do you have some error message? Please have a look here https://docs.shopware.com/en/shopware-5-en/tutorials-and-faq/debugging-error-messages-in-shopware#tweaks-for-the-config-php

**Comment by BenRoe** [comment]
I use [Lando with the LAMP receipt](https://docs.lando.dev/config/lamp.html) and Shopware 5.7 on OSX 10.15.7 

```
Fatal error: Uncaught Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException: 
The "swag_plugin_system.subscriber.template_registration" service or alias has been removed or inlined when the container 
was compiled. You should either make it public, or stop using the container directly and use dependency injection instead. in 
/app/app/vendor/symfony/dependency-injection/Container.php:275 Stack trace: #0 /app/app/vendor/symfony/dependency-injection/Container.php(231): 
Symfony\Component\DependencyInjection\Container->make('swag_plugin_sys...', 1) #1 /app/app/engine/Shopware/Components/DependencyInjection/Container.php(210): 
Symfony\Component\DependencyInjection\Container->get('swag_plugin_sys...', 1) #2 /app/app/engine/Shopware/Components/DependencyInjection/Container.php(134): 
Shopware\Components\DependencyInjection\Container->doLoad('swag_plugin_sys...', 'swag_plugin_sys...', 1) #3 /app/app/engine/Shopware/Components/ContainerAwareEventManager.php(191): 
Shopware\Components\DependencyInjection\Container->ge in /app/app/vendor/symfony/dependency-injection/Container.php on line 275
```

**Comment by mitelg** [comment]
hey @BenRoe 
this is a known issue and will be fixed to the final 5.7 release, where the subscriber will be set to public by default
